specification/compute/resource-manager/Microsoft.Compute/2017-10-15-preview/swagger/computeArtifactPublishingService.json

```diff
@@ -0,0 +1,870 @@
+        },
+        "classicComputeProfile": {
+          "$ref": "#/definitions/SharedVMImageClassicComputeProfile",
+          "description": "Properties of the image classic compute profile."
+        },
+        "purchasePlan": {
+          "$ref": "#/definitions/SharedVMImagePurchasePlan",
+          "description": "Properties of the image purchase plan."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string"
+        },
+      },
+      "required": [
+        "label",
+        "osType",
+		"identifier",
+		"classicComputeProfile"
+      ],
+      "description": "Shared VM image resource properties."
+    },
+    "SharedVMImageIdentifier": {
+      "properties": {
+        "publisher": {
+          "type": "string",
+          "description": "Publisher name for which the image is being published."
+        },
+        "offer": {
+          "type": "string",
+          "description": "Offer name for which the image is being published."
+        },
+        "sku": {
+          "type": "string",
+          "description": "Sku name for which the image is being published."
+        }
+      },
+      "required": [
+        "publisher",
+        "ofer",
+        "sku"
+      ],
+      "description": "Shared VM image identifier."
+    },
+    "SharedVMImageClassicComputeProfile": {
+      "properties": {
+        "imageType": {
+          "type": "string",
+          "description": "The image type.",
+          "enum": [
+            "VMImage",
+            "OSImage"
+          ],
+          "x-ms-enum": {
+            "name": "ImageTypes",
+            "modelAsString": false
+          }
+        },
+        "imageFamily": {
+          "type": "string",
+          "description": "ImageFamily for categorization in classic portal."
+        },
+        "iconUri": {
+          "type": "string",
+          "description": "IconUri of the image, must be a valid URI."
+        },
+        "smallIconUri": {
+          "type": "string",
+          "description": "Small thumbnail IconUri of the image, must be a valid URI."
+        },
+      },
+      "required": [
+        "imageType"
+      ],
+      "description": "Shared VM image classic compute profile."
+    },
+    "SharedVMImagePurchasePlan": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Purchase plan name."
+        },
+        "publisher": {
+          "type": "string",
+          "description": "Publisher name."
+        },
+        "product": {
+          "type": "string",
+          "description": "Product name."
+        }
+      },
+      "required": [
+      ],
+      "description": "Shared VM image purchase plan."
+    },
+	"SharedVMImageVersion": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SharedVMImageVersionProperties"
+        }
+      },
+      "allOf": [{
+          "$ref": "#/definitions/Resource"
```

Comment:
If this is not a tracked resource, then it wont have location/tags properties

Reply:
Shared VM Image and Version are tracked resource.